docs/deployment/cumulus-distribution.md

```diff
@@ -0,0 +1,55 @@
+---
+id: cumulus_distribution
+title: Using the Cumulus Distribution API
+hide_title: false
+---
+
+The Cumulus Distribution API is a set of endpoints that can be used to enable AWS Cognito authentication when downloading data from S3.
+
+## Configuring a Cumulus Distribution deployment
+
+The Cumulus Distribution API is included in the main [Cumulus](https://github.com/nasa/cumulus/tree/master/tf-modules/cumulus_distribution) repo. It is available as part of the `terraform-aws-cumulus.zip` archive in the [latest release](https://github.com/nasa/cumulus/releases).
+
+These steps assume you're using [the Cumulus Deployment Template](https://github.com/nasa/cumulus-template-deploy/blob/master/cumulus-tf/main.tf) but can also be used for custom deployments.
+
+To configure a deployment to use Cumulus Distribution:
+
+ 1. Remove or comment the "Thin Egress App Settings" in [the Cumulus Template Deploy](https://github.com/nasa/cumulus-template-deploy/blob/master/cumulus-tf/main.tf) and enable the Cumulus Distribution settings.
+ 2. Delete or comment the contents of [thin_egress_app.tf](https://github.com/nasa/cumulus-template-deploy/blob/master/cumulus-tf/thin_egress_app.tf) and the corresponding Thin Egress App outputs in [outputs.tf](https://github.com/nasa/cumulus-template-deploy/blob/master/cumulus-tf/outputs.tf). These are not necessary for a Cumulus Distribution deployment.
+ 3. Uncomment the Cumulus Distribution outputs in [outputs.tf](https://github.com/nasa/cumulus-template-deploy/blob/master/cumulus-tf/outputs.tf).
+ 4. Set the Cumulus Distribution variables in your `terraform.tfvars` (which is based on [terraform.tfvars.example](https://github.com/nasa/cumulus-template-deploy/blob/master/cumulus-tf/terraform.tfvars.example)). These include:
+
+  - cumulus_distribution_url
+  - csdap_client_id
+  - csdap_client_password
+  - csdap_host_url
+  - any others under the `Cumulus Distribution Variables` section in the example
+
+  5. Rename `cumulus-template-deploy/cumulus-tf/cumulus_distribution.tf.example` to `cumulus-template-deploy/cumulus-tf/cumulus_distribution.tf`.
```

Comment:
There's a lot of manual intervention here to switch between distribution APIs. You have to comment/delete a bunch of things and add a few others. We switch the `cumulus_distribution` module based on a conditional but that isn't as straightforward in all cases. It's almost certainly doable but it's a question of effort given how often people will make the switch. Open to conversation here as we could very probably be more elegant here.